src/approaches/oracle_approach.py

```diff
@@ -131,29 +138,38 @@ def pick_sampler(state: State, rng: np.random.Generator,
             ub = float(state.get(b, "pose") + state.get(b, "width")/2)
             ub = min(ub, 1.0)
         return np.array(rng.uniform(lb, ub, size=(1,)), dtype=np.float32)
+
```

Comment:
remove blank line (or, make it consistently be there across the file if you'd like)